onnxruntime/test/providers/tensorrt/tensorrt_basic_test.cc

```diff
@@ -5,6 +5,9 @@
 #include "test/providers/provider_test_utils.h"
 #include "test/framework/test_utils.h"
 #include "gtest/gtest.h"
+
+#if 0 // TODO: Make this work with TensorRT as a shared library
```

Comment:
>if [](start = 1, length = 2)

Resolve

Reply:
I don't know how... I need to talk to George and Xian about this as this code assumes providers are statically linked to the test code. It directly creates objects that only exist inside of the shared library.